docroot/modules/custom/sitenow_articles/sitenow_articles.module

```diff
@@ -556,6 +556,12 @@ function sitenow_articles_preprocess_block(&$variables) {
 
       break;
 
+    // @todo Move this to a more appropriate spot, if we may
+    //   want to use the gallery image field for other content.
+    case 'field_block:node:article:field_gallery_images':
+      $variables['attributes']['class'][] = 'grid--fivecol--20 photoswipe-gallery';
+
+      break;
```

Comment:
Wondering if others have thoughts on this. I'd begun setting up the gallery image field as pretty generic, where it may be reused in other content types (similar to `field_image`), but I'm not sure where best to place things like this. `layout_builder_custom` seems wrong, but so does this.